coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -2523,6 +2523,54 @@ def forward(self, x):
 
         self.run_compare_torch(input_shape, model, backend=backend)
 
+    @pytest.mark.parametrize("backend", backends)
+    def test_expand_dynamic_shape0(self, backend):
+        class TestModel(nn.Module):
+            def forward(self, x):
+                return x.expand(x.shape[1], x.shape[1])
+
+        self.run_compare_torch(torch.arange(20).reshape((1, 20)),
+                               TestModel(),
+                               input_as_shape=False,
+                               converter_input_type=[TensorType(shape=[1, ct.RangeDim()])],
+                               backend=backend)
+
+    @pytest.mark.parametrize("backend", backends)
+    def test_expand_dynamic_shape1(self, backend):
+        class TestModel(nn.Module):
+            def forward(self, x):
+                return x.expand(x.shape[0], 1, x.shape[-1], x.shape[-1])
+
+        self.run_compare_torch(torch.arange(20).reshape((1, 20)),
+                               TestModel(),
+                               input_as_shape=False,
+                               converter_input_type=[TensorType(shape=[1, ct.RangeDim()])],
+                               backend=backend)
+
+    @pytest.mark.parametrize("backend", backends)
+    def test_expand_dynamic_shape2(self, backend):
+        class TestModel(nn.Module):
+            def forward(self, x):
+                return x.expand(x.shape[-1], 1, x.shape[-1], x.shape[-1])
+
+        self.run_compare_torch(torch.arange(20).reshape((1, 20)),
+                               TestModel(),
+                               input_as_shape=False,
+                               converter_input_type=[TensorType(shape=[1, ct.RangeDim()])],
+                               backend=backend)
+
+    @pytest.mark.parametrize("backend", backends)
+    def test_expand_dynamic_shape_and_broadcast(self, backend):
```

Comment:
why is this test has a different name?
For instance, `test_expand_dynamic_shape2` is also testing `expand and broadcast` isn't it?

Reply:
I've standardized the test names.